sdwan/utils.go

```diff
@@ -72,3 +72,140 @@ func interfaceToStrList(data interface{}) []string {
 
 	return strList
 }
+
+func belongsToNTP(lookup string) bool {
+	switch lookup {
+	case
+		"vedge-1000",
+		"vedge-2000",
+		"vedge-cloud",
+		"vedge-5000",
+		"vedge-ISR1100-6G",
+		"vedge-100-B",
+		"vedge-ISR1100-4G",
+		"vedge-100",
+		"vsmart",
+		"vedge-ISR1100-4GLTE",
+		"vedge-100-WM",
+		"vmanage",
+		"vbond",
```

Comment:
Are we managing vmanage, vsmart and vbond device types? if not please remove them.

Reply:
removed vmanage, vsmart and vbond device types.